Makefile

```diff
@@ -206,67 +206,67 @@ docker-ci-integration:
 
 .PHONY: docker-build-collection
 docker-build-collection:
```

Comment:
why not just `git checkout origin/master -- Makefile` since there are no real changes except spaces?

Reply:
I thought the fix to spaces are actually good :D